src/PrestaShopBundle/Resources/config/services/bundle/controller.yml

```diff
@@ -8,6 +8,8 @@ services:
         calls:
             - [setLogger, ['@logger']]
             - [setContainer, ['@service_container']]
+        tags:
+          - { name: prestashop_core.controllers }
```

Comment:
Any reason to use `prestashop_core` instead of `prestashop.core`>

Reply:
No, I just wanted to make unambiguous the fact these are the core controllers (to distinguish from module controllers or 3rd party controllers). I chose the name in 2 seconds, it can be improved indeed 👍